R/resolve.R

```diff
@@ -12,7 +12,9 @@
 #' @param na.rm Should NAs be removed? True by default.
 #' @return A single scalar or vector of dates
 #' @examples
-#' d <- as_messydate("2014-01-01..2014-01-31")
+#' d <- as_messydate(c("2008-03-25", "?2012-02-27", "2001-01?", "2001~",
```

Comment:
Lintr notes for snake_case style relate to functions having a "." instead of a "_", I think... Just letting you know, but it is not a big deal and I do not think it is a reason for a change unless we want to use "_" consistently across the package for all functions...